make/istio/stop

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+: "${GIT_ROOT:=$(git rev-parse --show-toplevel)}"
+source "${GIT_ROOT}/make/include/has_namespace"
+source "${GIT_ROOT}/make/include/has_release"
+source "${GIT_ROOT}/make/include/defaults"
+
+printf "Stopping and cleaning up %s release...\\n" "${ISTIO_HELM_RELEASE}"
+
+if has_release "${ISTIO_HELM_RELEASE}"; then
+  helm delete --purge "${ISTIO_HELM_RELEASE}"
+fi
+
+if has_namespace "${ISTIO_NAMESPACE}"; then
+  kubectl delete namespace "${ISTIO_NAMESPACE}"
+fi
+
+# Delete Custom Resource Definitions.
+custom_resource_definitions=$(kubectl get customresourcedefinitions -o=go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}' | grep "istio.io" | cat)
```

Comment:
I don't think we need the `| cat` at the end?
(I also find it annoying that there isn't a built-in `strings.Contains` function)